recipes/simple-build-system/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = "0.0.20" %}
+
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: simple-build-system
+  version: {{ version }}
+
+source:
+  url: https://github.com/mctools/simplebuild/archive/v{{ version }}.tar.gz
+  sha256: 5fb9950d95146299e5ce6dc8e0bf15ca3e10e13e2addb9fd5a4818add29a6878
+
+build:
+  skip: True  # [win or py<39]
+  number: 0
+  script: python -m pip install --no-deps -vv .
+  entry_points:
+    - sb = _simple_build_system._cli:main
+    - unwrapped_simplebuild = _simple_build_system._cli:unwrapped_main
+    - sbenv = _simple_build_system._cli:sbenv_main
+    - sbrun = _simple_build_system._cli:sbrun_main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tomli  # [py<311]
+    - pybind11 >=2.10.4
+    - cmake >=3.24.2
+    - make
+    - numpy
+
+test:
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Can you also test that at least one entry point works?
```suggestion
    - {{ compiler('cxx') }}
    - pip
  commands:
    - pip check
    - sb --help
```

---

The tests in `run_test.sh` actually exercise both the `sb` and `sbenv` entry points. Or did you mean something else?

---

To elaborate: `sb --help` is already in `run_test.sh`, but `pip check` is not. Should I add `pip check` to `run_test.sh` as well?

I am afraid to simply commit your suggestion, since I thought (perhaps wrongly?) that one could either specify test commands in `meta.yaml` *or* have a `run_test.sh`, but not both?

---

I have now pushed a change which adds `pip check` to run_test.sh as well, and which removes the duplicated (and I guess misleading) import line from `meta.yaml`. Instead all the actual tests are now in `run_test.sh`, and just the test requirements are listed in `meta.yaml`. I assume this resolves this.